style.css

```diff
@@ -471,3 +400,224 @@ form textarea {
   height: 114px;
   width: 327px;
 }
+
+.main-section {
+  padding: 0 30px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 15px;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  background-image: url("./images/bg.png");
+  background-repeat: no-repeat;
+  height: 400px;
+  justify-content: flex-end;
+  gap: 10px;
+}
+
+.tech-ul {
+  list-style: none;
+  display: flex;
+  width: 70%;
+  gap: 18px;
+  padding-left: 15px;
+}
+
+.tech-ul-li {
+  padding: 10px;
+  background-color: rgb(255, 255, 255, 25%);
+  color: #fff;
+}
+
+.card-paragraph {
+  color: #fff;
+  font-size: 19px;
+  padding-left: 15px;
+  width: 70%;
+}
+
+.card-header {
+  color: #fff;
+  font-size: 28px;
+  width: 70%;
+  padding-left: 15px;
+}
+
+@media screen and (max-width: 767px) {
+  .nav-list {
+    display: none;
+  }
+
+  .line {
+    display: none;
+  }
+
+  .works .placeholder {
+    padding-top: 60px;
+    margin: 0;
+  }
+}
+
+@media (min-width: 768px) {
+  .top-section {
+    height: 72px;
+    padding: 26px 100px;
+  }
+
+  .menu-wrapper {
+    display: none;
+  }
+
+  .nav-list {
+    font-family: "Inter", sans-serif;
+    list-style: none;
+    display: flex;
+    align-content: space-between;
+  }
+
+  .nav-item {
+    margin-left: 32px;
+  }
+
+  .nav-item a {
+    text-decoration: none;
+    color: #fff;
+  }
+
+  .hero {
+    background: #1c1a19 url(./images/HeaderIcons/desktop_illustation.png);
+    background-repeat: no-repeat;
+    background-size: 380px;
+    background-position-x: right;
+    background-position-y: -44px;
+    display: flex;
+    flex-direction: row-reverse;
+    gap: 40px;
+    justify-content: flex-end;
+  }
+
+  .social {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 12px;
+  }
+
+  .heroSocial {
+    display: flex;
+  }
```

Comment:
Kindly align the text as shown in the Figma design, You can try to increase the width of the text-container ✊ 

Your Project
![image](https://user-images.githubusercontent.com/66112760/161404415-f46d29d3-20dc-429e-9197-3cfa8f1f0fac.png)

Figma design template
![image](https://user-images.githubusercontent.com/66112760/161404456-7cc7b867-ecfd-4d90-853c-f57d5c2d3e01.png)